### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by rbohl**
Calendar documentation, for SME review.

### CODE REVIEW

File: discover/portal/articles/230-calendar/00-calendar-intro.markdown [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+![Options](../../images/icon-app.options.png) in the widget's header and click
+*Configuration*.<!--deleted references to tabs because there isn't any material
+here on the Display Settings and RSS tabs -NR-->
+
+![Figure 1: The Setup &rarr; User Settings tab provides the options you need to get started quickly.](../../images/new-calendar-configuration.png)
+
+From the *User Settings* tab, you can customize your calendar's default
+view and settings. 
+
+**Time Format:** Choose *AM/PM* or to *24 Hour*. *AM/PM* is the default; with
```
Comment (natocesarrego) [inline discover/portal/articles/230-calendar/00-calendar-intro.markdown]: What about the _Locale_ value?

File: discover/portal/articles/230-calendar/01-using-the-calendar-widget.markdown [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+At the bottom of the *Edit Event* form, there are several collapsed sections:
+*Details*, *Invitations*, *Reminders*, *Categorization*, and related *Related
+Assets*.
+
+#### Details
+
+In the Details collapsible section, you can change the calendar to which the
+event belongs, enter a location, and configure the event's permissions.
+
+Giving a user permission to add, delete, or update discussion allows them to
```
Comment (natocesarrego) [inline discover/portal/articles/230-calendar/01-using-the-calendar-widget.markdown]: I've checked this section on master and I've not found the permissions configuration.

File: discover/portal/articles/230-calendar/02-calendar-resources-and-porting.markdown [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+    Under *Content*, decide which content types (calendars, resources, and
+    events) to import, and whether to include comments and ratings.
+
+    Choose whether to import permissions and deletions, and decide whether to
+    delete your widget's existing data before the import.
+
+5.  In the collapsible *Update Data* section, choose how data will be updated.
+
+    **Mirroring:** The data will be imported along with a reference to its
```
Comment (natocesarrego) [inline discover/portal/articles/230-calendar/02-calendar-resources-and-porting.markdown:95]: The UI names it as _Mirror_ only. 😉

**Review by natocesarrego — COMMENTED** [review]
Pull request reviewed! This is the last one of our User Guide! 🎉 🎉 😄

File: discover/portal/articles/230-calendar/00-calendar-intro.markdown (hunk 1 above)
Comment (rbohl) [reply discover/portal/articles/230-calendar/00-calendar-intro.markdown]: @natocesarrego  Good catch. Locale looks like the default, right?

Is this how it works? I'm guessing here:
>_Locale_ is a dynamic setting that chooses whether to display the time in _AM/PM_ or _24 Hour_ format, based on the preferences set by the User's locale.

File: discover/portal/articles/230-calendar/01-using-the-calendar-widget.markdown (hunk 2 above)
Comment (rbohl) [reply discover/portal/articles/230-calendar/01-using-the-calendar-widget.markdown]: hmm, looks to be present on 7.1 CE GA1

File: discover/portal/articles/230-calendar/00-calendar-intro.markdown (hunk 1 above)
Comment (rbohl) [reply discover/portal/articles/230-calendar/00-calendar-intro.markdown]: After testing it seems I am correct. But let me know if I am missing something.

**Comment by rbohl** [comment]
@natocesarrego see my comments above and my review edits here: https://github.com/rbohl/liferay-docs/commit/8181ae174791a6b2f3426f66ee5f3f7c6a55dad8

and let me know if this looks good to you.
Thanks!

File: discover/portal/articles/230-calendar/00-calendar-intro.markdown (hunk 1 above)
Comment (natocesarrego) [reply discover/portal/articles/230-calendar/00-calendar-intro.markdown]: Yeah, _Locale_ is the default value. Besides, you're correct regarding the behavior mentioned by you. 😄

**Comment by natocesarrego** [comment]
Green light for me! Good job!

**Comment by rbohl** [comment]
sent for publication: https://github.com/sez11a/liferay-docs/pull/3543

**Pull request merged**